src/components/Callout/Callout.Props.ts

```diff
@@ -81,3 +95,8 @@ export interface ILink {
    */
   url: string;
 }
+
+export enum CalloutBackgroundColor {
+  white,
+  blue
```

Comment:
Callout is generic, but I'd like to support various background colors. I find that these "close to the metal" components are more utilitarian, and uses values like "left/right" or "black/white" whereas the composite patterns that put them together use words that relate to the content inside of them.

I.E. Callout doesn''t have a clue that teaching bubbles should be blue. That should be up to the teaching bubble to decide. The callout just needs to make it possible to change to a blue background.

If you have suggestions on other words to use, I'm happy to use them. If there is some primary/secondary/tertiary color relation I'd be happy to use that.